CHANGELOG.md

```diff
@@ -1,3 +1,125 @@
+### citus v10.0 (February 16, 2021) ###
+
+* Adds shard rebalancer
+
+* Adds `pg_catalog.time_partitions` view for simple (time) partitions and their access methods
+
+* Adds `alter_old_partitions_set_access_method` UDF to compress old partitions
+
+* Adds cascade option to `undistribute_table` UDF for foreign key connected relations
+
+* Adds `pg_catalog.citus_shards`, `pg_catalog.citus_shards_on_worker` and `pg_catalog.citus_shard_indexes_on_worker` views
+
+* Adds `public.citus_tables` view
+
+* Adds `remove_local_tables_from_metadata` UDF for local tables automatically added to metadata when defining foreign keys between reference tables and local tables
+
+* Adds support for `CREATE INDEX` commands without index name on citus tables
+
+* Adds support for `CREATE TABLE ... USING table_access_method` statements
+
+* Adds support for co-located/recurring correlated subqueries
+
+* Adds support for co-located/recurring sublinks in the target list
+
+* Adds support for local distributed/subquery joins 
+
+* Adds support for local execution in `INSERT..SELECT` commands that perform re-partitioning
+
+* Adds `alter_distributed_table` and `alter_table_set_access_method` UDFs
+
+* Allows distributed table creation immediately after CREATE EXTENSION citus
+
+* Automatically adds coordinator to the metadata before the first distributed table created
+
+* Improves performance in some subquery pushdown cases
+
+* Introduces adaptive connection management for local nodes
+
+* Propagates `ALTER TABLE .. SET LOGGED/UNLOGGED` statements
+
+* Propagates `ALTER INDEX .. SET STATISTICS` statements
+
+* Propagates `ALTER SCHEMA RENAME` statements
+
+* Propagates `CREATE STATISTICS` statements across workers.
+
+* Propagates `DROP STATISTICS` statements across the workers
+
+* Propagates all types of `ALTER STATISTICS` statements.
+
+* Propagates columnar table settings for distributed tables
+
+* Removes `citus.binary_master_copy_format` GUC
+
+* Removes `citus.expire_cached_shards` GUC
+
+* Removes `citus.large_table_shard_count` GUC
+
+* Removes `citus.max_assign_task_batch_size` GUC
+
+* Removes `citus.max_running_tasks_per_node` GUC
+
+* Removes `citus.max_task_string_size` GUC
+
+* Removes `citus.max_tracked_tasks_per_node` GUC
+
+* Removes `citus.sslmode` GUC
+
+* Removes `citus.task_tracker_delay` GUC
+
+* Removes the word 'master' from Citus UDFs
+
+* Removes unused UDF `mark_tables_colocate`
+
+* Removes `upgrade_to_reference_table` UDF
+
+* Renames 'master' to 'distributed' for columns of `citus_dist_stat_activity`
+
+* Renames 'master' to 'distributed' for columns of `citus_worker_stat_activity`
+
+* Avoids establishing new connections until the previous ones are finalized
+
+* merge cstore into the citus repo
```

Comment:
I think we should re-phrase columnar entry and move it to top, maybe `Add columnar tables` or something ?

Reply:
Yes, sure we should do that. This should be the first item in the release notes. Maybe, use the sentence from the wiki: https://github.com/citusdata/citus/tree/master/src/backend/columnar#introduction
```Adds support for per-table option for columnar storage```

Reply:
maybe `Adds columnar tables with columnar access method`